apps/mobile/fastlane/test-release-please.sh

```diff
@@ -0,0 +1,6 @@
+release-please release-pr \
+--token=${GITHUB_TOKEN} \
+--repo-url=leather-io/mono \
+--target-branch=chore/LEA-1844/clean-fastlane \
```

Comment:
This target-branch shouldn't be hardcoded right?

Reply:
this is just a testing script - I plan to delete this before merging.